apps/civil/civil-sdt/ithc-01.yaml

```diff
@@ -7,4 +7,5 @@ spec:
   values:
     java:
       environment:
+        PURGE_SCHEDULE_CRON: 0 15 0 * * 1,2,3
         IDAM_API_URL: https://idam-api.platform.hmcts.net
```

Comment:
Remove the IDAM_API_URL environment variable.  It's only needed in the prod environment flux config files (because the value in the prod environment is different to the other environments).

Reply:
done